ansible/spatial.yml

```diff
@@ -5,7 +5,7 @@
     - java
     - tomcat
     - webserver
-    - {role: postgresql, pg_service: 'postgresql-9.6', pg_version: '9.6', postgis_version: '2.4'}
+    - postgresql
```

Comment:
instead of removing the version, why not add `spatial_pg_version` and `sptaial_postgis_version` and then override them in the relevant inventory?